src/Microsoft.AspNetCore.StaticFiles/Infrastructure/SharedOptionsBase.cs

```diff
@@ -11,8 +10,7 @@ namespace Microsoft.AspNetCore.StaticFiles.Infrastructure
     /// <summary>
     /// Options common to several middleware components
     /// </summary>
-    /// <typeparam name="T">The type of the subclass</typeparam>
-    public abstract class SharedOptionsBase<T>
+    public abstract class SharedOptionsBase
```

Comment:
I'm sure it was due to the them needing to implement IOptions of themselves or some other grossness :)